temp/gitbasics.txt

```diff
@@ -0,0 +1,29 @@
+1.git config --global user.name "username"
+2.git config --global user.email "email"
+3.git init
+4.git clone
+5.git status
+6.git add
+7.git push -u origin master   or git push origin branchname
+8.git pull
```

Comment:
Example: git pull > Pull the current changes in the origin/branch. This command will update the current commits.

Usage is missing, please update with an explanation.

Reply:
OKAY I GOT IT.